Remove extra warning flags that are not understood by old GCC versions.
https://github.com/weechat/weechat/issues/891#issuecomment-1693308171
--- CMakeLists.txt.orig	2024-04-07 11:37:29.000000000 -0500
+++ CMakeLists.txt	2024-05-17 23:06:30.000000000 -0500
@@ -33,8 +33,6 @@
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fsigned-char -fms-extensions -Wall -Wextra")
 if (CMAKE_C_COMPILER_ID STREQUAL "GNU")
   # extra options specific to gcc/g++
-  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wformat-overflow=2 -Wformat-truncation=2")
-  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wformat-overflow=2 -Wformat-truncation=2")
 endif()
 
 # version
